src/BaseLevelScene.js

```diff
@@ -45,8 +45,11 @@ class BaseLevelScene extends Phaser.Scene {
         this.input.keyboard.on('keydown', (event) => {
             switch (event.key) {
                 case 'Enter':
-                    this.catLoosesLive();
-                    this.startNextLevel(false, 1);
+                    // Only when in start screen or end screen
+                    if (this.currentSceneIndex == 0 || this.currentSceneIndex == this.scenes.length-1) {
```

Comment:
I would suggest to make it a bit more clear what the intention is here (then maybe you even don't need the comment explaining what the code does ;) )

```suggestion
                    if (
                        this.scenes[this.currentSceneIndex] === 'StartScene'
                        || this.scenes[this.currentSceneIndex] === 'EndScene'
                    ) {
```